### PR TITLE
Upgrade Ruby to 3.1.6, 3.3.2

### PR DESCRIPTION
**Pull request by manuelvanrijn**
New ruby releases are out

Release notes: 
* https://www.ruby-lang.org/en/news/2024/05/29/ruby-3-1-6-released/
* https://www.ruby-lang.org/en/news/2024/05/30/ruby-3-3-2-released/

### CODE REVIEW

**Comment by manuelvanrijn** [comment]
Hi @FooBarWidget, could you please take a look at this? Thanks! 😄

File: config.yml [hunk 1]
```diff
@@ -12,13 +12,13 @@ ruby:
   ## then be sure to bump the corresponding `package_revision`!
   minor_version_packages:
     - minor_version: '3.3'
-      full_version: '3.3.1'
```
Comment (FooBarWidget) [inline config.yml:15]: For minor version packages, please bump the package_revision as well.

**Review by FooBarWidget — CHANGES_REQUESTED** [review]

File: config.yml (hunk 1 above)
Comment (manuelvanrijn) [reply config.yml:15]: Ahh, sorry about that. I've pushed the changes

**Comment by FooBarWidget** [comment]
Looks good. CI passed so this'll be released in a couple of hours.

**Pull request merged**